README.md

```diff
@@ -43,8 +43,8 @@ import Glide
 let app = Application()
 
 // 3. Add a route.
-app.get("/hello") { _, response in
-  response.send("Hello, world!")
+app.get("/hello") { _, _ in
+  .send("Hello, world!")
```

Comment:
Implicit return make for some nice syntactic sugar with this version of the API 😉